### PR TITLE
Radios - Fix stop working only for player stopping it

### PR DESCRIPTION
**Pull request by Mike-MF**
**When merged this pull request will:**
- Should fix the source not being deleted for everybody.

### CODE REVIEW

File: addons/radios/XEH_postInitClient.sqf [hunk 1]
```diff
@@ -6,6 +6,12 @@
     _vehicle setVariable [QGVAR(source), _source];
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(stop), {
+    params ["_vehicle", "_source"];
+    deleteVehicle _source;
```
Comment (jonpas) [inline addons/radios/XEH_postInitClient.sqf:11]: Check if `_source` exists, it will not for JIP clients.

Comment (Mike-MF) [reply addons/radios/XEH_postInitClient.sqf:11]: That's done just before the event is called in `fnc_stop`

Comment (jonpas) [reply addons/radios/XEH_postInitClient.sqf:11]: No, it should be on each client, as some might have it, some not.

File: addons/radios/XEH_postInitClient.sqf [hunk 2]
```diff
@@ -6,6 +6,14 @@
     _vehicle setVariable [QGVAR(source), _source];
 }] call CBA_fnc_addEventHandler;
 
+[QGVAR(stop), {
+    params ["_vehicle"];
+    private _source = _vehicle getVariable [QGVAR(source), objNull];
+    if (isNull _source) exitWith {}; // Handles Killed & JIP
+    deleteVehicle _source;
+    _vehicle setVariable [QGVAR(source), objNull];
```
Comment (jonpas) [inline addons/radios/XEH_postInitClient.sqf:14]:
```suggestion
    params ["_vehicle"];

    private _source = _vehicle getVariable [QGVAR(source), objNull];
    if (isNull _source) exitWith {}; // Handles Killed & JIP

    deleteVehicle _source;
    _vehicle setVariable [QGVAR(source), objNull];
```

**Review by jonpas — APPROVED** [review]

**Pull request merged**